### ISSUE TITLE
Style guide amendment request

### ISSUE THREAD

**Issue by kurahaupo** [Libera-Chat/libera-chat.github.io#29]
Documentation in MD or HTML is not restricted to a single monospace font, so I suggest that we should use _`parameter`_ rather than `<parameter>`, in line with normal typesetting for books, manuals, etc.

This is separate from denoting optional and/or related items, which can be literals, parameters, or non-terminals. The [ ] and … should be in the ordinary run typeface rather than the fixed-pitch font used for code examples.

So a more complete example would look like:
[`--option`_`value`_ …]
or
`/msg chanserv register`_`#channel`_
`/join`_`#channel`_[`,`_`#channel`_ …]

**Comment by swantzter** [comment]
I don't agree partly because I think this won't look great rendered (I guess unless we put in considerable work to create a "command group" thing) but to me it starts looking like things are separate rather than one thing
`<parameter>` and `[parameter]` are also in line with how atheme shows things when you speak to services on IRC, and I think that consistency is important, it's also a relatively established way of doing it

**Comment by kurahaupo** [comment]
I've never seen services generate mIRC coded text, so it doesn't surprise me that it's not used there.

This would be more important in contexts where `<>[]` are syntactic elements being described, which then makes them a poor choice for documentation meta-syntax. They are particularly problematic in documentation for programming languages, with `[]` for array indices and `<>` for comparisons and/or file-descriptor redirections.

It is unfortunate that markdown doesn't easily let one put italicized elements inside a code block, so you have to do the block entirely with HTML instead. But at least that's allowed, like: <tt>example code with <i>parameter</i> and <b>keyword</b> included</tt>.
